client-rpc/src/rpc/websocket_rpc.rs

```diff
@@ -0,0 +1,126 @@
+use crate::rpc::websocket_core::WebsocketCore;
+use chain_core::state::account::StakedStateAddress;
+use client_common::tendermint::Client;
+use client_common::{PrivateKey, PublicKey, Storage};
+use client_index::BlockHandler;
+use futures::future::Future;
+use futures::sink::Sink;
+use futures::stream::Stream;
+use futures::sync::mpsc;
+use mpsc::Sender;
+use std::thread;
+use websocket::result::WebSocketError;
+use websocket::{ClientBuilder, OwnedMessage};
+
+pub const CONNECTION: &str = "ws://localhost:26657/websocket";
```

Comment:
This would ideally be configurable rather than hard-coded -- e.g. via a command-line argument or as an environment variable.

Reply:
i will add as runtime argument

Reply:
ok